.travis.yml

```diff
@@ -10,23 +10,55 @@ script:
   - script/build
 xcode_workspace: ReactiveCocoa.xcworkspace
 matrix:
+  allow_failures:
+    - xcode_scheme: ReactiveCocoa-iOS
+      osx_image: xcode8
```

Comment:
It seems like it's consistently failing? 😕 Is this just a CI issue or does it fail locally too?